components/filter/filter-tags.js

```diff
@@ -91,25 +56,17 @@ class FilterTags extends RtlMixin(LocalizeCoreElement(LitElement)) {
 				</d2l-tag-list-item>
 			`;
 		}));
-		let filters = html``;
-		if (numActiveFilters === 0) filters = html`<span class="d2l-filter-tags-none-label d2l-body-compact">${this.localize('components.filter.noActiveFilters')}</span>`;
-		else {
-			filters = html`
-				<d2l-tag-list
-					clearable
-					clear-focus-timeout="${CLEAR_TIMEOUT}"
-					@d2l-tag-list-clear="${this._clearFiltersClicked}"
-					description="${this.label || this.localize('components.filter.activeFilters')}">
-					${tagListItems}
-				</d2l-tag-list>
-			`;
-		}
+
+		if (numActiveFilters === 0) return nothing;
```

Comment:
Already discussed with Jeff whether we want any offscreen text here, but we decided this would be the cleaner experience

Reply:
Would it be worth rendering some whitespace that is the same height as a filter-tag instead of nothing so that when filters are added or removed the page doesn't jitter up or down?

Reply:
Yeah we also talked about this - Jeff specifically didn't want to save space for it.  Ideally he wanted an animation to make that jitter a bit smoother, but after investigating I'm going to throw that in a separate story.  It's not very straightforward because `tag-list` isn't responsible for actually removing the `tag-list-item`s, and the removal happens immediately.  I think we need to store the height, trigger an animation to no height, and _then_ remove the tags from the `slot`.

Reply:
Added this as [US146439](https://rally1.rallydev.com/#/15545167705ud/search?detail=%2Fuserstory%2F673054076481&forbidFormattedIdRedirect=true&keywords=filter%20tag)